templates/template-angular/angular.json.lte

```diff
@@ -8,32 +8,43 @@
   "projects": {
     "{% package_name %}": {
       "projectType": "application",
+      "schematics": {
+        "@schematics/angular:component": {
+          "style": "scss"
+        }
+      },
```

Comment:
why force scss? is the new default for angular?

Reply:
My main goal is to stick to what angular official tool generates (it is logical that this is the most used angular template), to reduce any friction, and let users just focus on tauri APIs example included in the template.